app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -28,30 +30,42 @@ IUSE=""
 
 DEPEND=">=dev-lang/go-1.3
 	app-arch/cpio
-	sys-fs/squashfs-tools"
+	sys-fs/squashfs-tools
+	dev-perl/Capture-Tiny"
 RDEPEND="!app-emulation/rocket"
 
-src_unpack() {
-	local cache="${S}/stage1/rootfs/usr_from_coreos/cache"
+src_configure() {
+	local myeconfargs=(
+		--with-stage1=host
```

Comment:
`--with-stage1=host` pulls from `/` at run-time (when commands like `rkt run ...` are executed), not at build-time. So the builder architecture or $ROOT should not be relevant.

But I agree that the pre-built `coreos` is more tested for now, so safest indeed.

@crawford yes, with `host`, rkt 1.2.3 will potentially run differently depending on the underlying CoreOS version: when the host switches to a new version of systemd, rkt containers will automatically use the new systemd even if rkt is not updated. This puts more burden on the distribution...